hamilton/function_modifiers/validation.py

```diff
@@ -38,13 +39,21 @@ def transform_node(
         validators = self.get_validators(node_)
         validator_nodes = []
         validator_name_map = {}
+        validator_name_count = defaultdict(int)
```

Comment:
OK, so I realized, this won't work with multiple `@check_output_custom` decorators... Which means that we should error out but really, just add that to the docs for now. That they have to be in the same decorator. It's also just strictly better.

Reply:
it never did work. added comment in docs.